mojo/public/cpp/bindings/lib/buffer.h

```diff
@@ -130,7 +130,7 @@ class COMPONENT_EXPORT(MOJO_CPP_BINDINGS_BASE) Buffer {
 // consistent size. Used to track down the reason for mismatched message sizes
 // when replaying.
 struct AutoRecordReplayAssertBufferAllocations {
-  AutoRecordReplayAssertBufferAllocations();
+  AutoRecordReplayAssertBufferAllocations(const char* label = "");
```

Comment:
maybe have a name indicating this should be an issue number rather than the more generic "label" ?